tests/integration/targets/aws_ses_rule_set/defaults/main.yaml

```diff
@@ -1,6 +1,6 @@
 ---
-default_rule_set: "{{ resource_prefix }}-default-rule-set"
-second_rule_set: "{{ resource_prefix }}-second-rule-set"
+default_rule_set: "{{ resource_prefix[:32] }}-default-rule-set"
+second_rule_set: "{{ resource_prefix[:32] }}-second-rule-set"
```

Comment:
is [:32] going to be unique?  ie.  Will there only be one playbook running at any given time using this shortened resource prefix?

I suspect that the "azp-5268-1" part is all that's needed for the prefix to be "unique" and the rest makes it easier to tell what was being tested, but I don't know the AZP pieces yet